mllib/src/main/resources/org/apache/spark/ml/feature/stopwords/english.txt

```diff
@@ -149,5 +149,58 @@ shan
 shouldn
 wasn
 weren
-won
 wouldn
```

Comment:
I think we should remove the spurious stems; the info on this PR suggests they aren't supposed to be there.